azkaban-common/src/main/java/azkaban/executor/JdbcExecutorLoader.java

```diff
@@ -44,20 +39,17 @@
   private final FetchActiveFlowDao fetchActiveFlowDao;
   private final AssignExecutorDao assignExecutorDao;
   private final NumExecutionsDao numExecutionsDao;
-  private EncodingType defaultEncodingType = EncodingType.GZIP;
 
   @Inject
-  public JdbcExecutorLoader(final Props props, final CommonMetrics commonMetrics,
-      final ExecutionFlowDao executionFlowDao,
-      final ExecutorDao executorDao,
-      final ExecutionJobDao executionJobDao,
-      final ExecutionLogsDao executionLogsDao,
-      final ExecutorEventsDao executorEventsDao,
-      final ActiveExecutingFlowsDao activeExecutingFlowsDao,
-      final FetchActiveFlowDao fetchActiveFlowDao,
-      final AssignExecutorDao assignExecutorDao,
-      final NumExecutionsDao numExecutionsDao) {
-    super(props, commonMetrics);
+  public JdbcExecutorLoader(final ExecutionFlowDao executionFlowDao,
```

Comment:
The format seems to be different from the google style. Did you import and activate the new style config? See the readme.

Reply:
I should be using Google Style. I just checked the details of the style. It should allow multiline when method declaration parameter is too long. We may check together to make sure google style is working.